Lib/Protocols/IdPOP3.pas

```diff
@@ -587,6 +587,18 @@ procedure TIdPOP3.SetSASLMechanisms(AValue: TIdSASLEntries);
   FSASLMechanisms.Assign(AValue);
 end;
 
+procedure TIdPOP3.SendUTF8IfAdvertised;
+var
+  Capa : string;
+begin
+  for Capa in FCapabilities do
+    if TextStartsWith(Capa, 'UTF8 ') then

```

Comment:
The `UTF8` capability has an optional `USER` argument. If that argument is not present, the trailing space is likely to also not be present, which would cause this `TextStartsWith()` call to return False.  You can use `IdGlobal.Fetch()` to retrieve the first word before then comparing it.